container_registry/container_analysis/src/sample/samples.go

```diff
@@ -10,14 +10,18 @@ import (
 	"sync"
 	"time"
 
-	containeranalysis "cloud.google.com/go/devtools/containeranalysis/apiv1beta1"
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1beta1"
 	pubsub "cloud.google.com/go/pubsub"
 	"golang.org/x/net/context"
 	"google.golang.org/api/iterator"
 	grafeaspb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/grafeas"
 	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/vulnerability"
 )
 
+func main() {
```

Comment:
We should switch the package name to something else - snippets shouldn't have a main.

Reply:
Done